web/cobrands/shropshire/assets.js

```diff
@@ -18,6 +18,7 @@ var defaults = {
         }
     },
     asset_type: 'spot',
+    central_asset_id: 'CentralAssetId',
```

Comment:
the key is asset_id_field...

For completeness, presumably the traffic signal areas layer needs a different asset_id_field to match?